classes/CacheManager.php

```diff
@@ -99,43 +95,21 @@ public function putMatch(string $cacheKey, ?RedirectRule $matchedRule = null): ?
         if ($matchedRuleToDate instanceof Carbon) {
             $minutes = $matchedRuleToDate->diffInMinutes(Carbon::now());
 
-            $this->cache->tags(self::CACHE_TAG_MATCHES)
-                ->put($cacheKey, $matchedRule, $minutes);
+            $this->cache->put(self::CACHE_TAG_MATCHES . '.' . $cacheKey, $matchedRule, $minutes);
         } else {
-            $this->cache->tags(self::CACHE_TAG_MATCHES)
-                ->forever($cacheKey, $matchedRule);
+            $this->cache->forever(self::CACHE_TAG_MATCHES . '.' . $cacheKey, $matchedRule);
         }
 
         return $matchedRule;
     }
 
     public function cachingEnabledAndSupported(): bool
```

Comment:
@jaxwilko should we perhaps make this and the method below static?

Reply:
`cachingEnabledAndSupported` ? 

I have no strong opinion, however it's not the style that this code has been written in

Reply:
Meh, I don't really care about making it static at this point, I already don't really like the coding style of the entire plugin so I'd rather just be consistent with what there is right now and look at refactoring it later if there's a need to.

Reply:
It's more not having to instantiate the manager just to find out if it can be used or not. But it's already merged... so meh :P